doc/classes/Camera3D.xml

```diff
@@ -184,7 +184,7 @@
 			The distance to the far culling boundary for this camera relative to its local Z axis. Higher values allow the camera to see further away, while decreasing [member far] can improve performance if it results in objects being partially or fully culled.
 		</member>
 		<member name="fov" type="float" setter="set_fov" getter="get_fov" default="75.0">
-			The camera's field of view angle (in degrees). Only applicable in perspective mode. Since [member keep_aspect] locks one axis, [member fov] sets the other axis' field of view angle.
+			The camera's field of view angle (in degrees). Only applicable in perspective mode. Since [member keep_aspect] locks one axis, [member fov] sets the other axis's field of view angle.
 			For reference, the default vertical field of view value ([code]75.0[/code]) is equivalent to a horizontal FOV of:
```

Comment:
Do you remember that discussion regarding [overly long compound words](https://github.com/godotengine/godot-contributing-docs/pull/6#discussion_r2399459339)? This contains a similar tongue-twister.
Also, for consistency, as this very sentence later shortens "Field of view" into "FOV":
```suggestion
			For reference, the default angle for the vertical FOV ([code]75.0[/code]) is equivalent to a horizontal FOV of:
```